app/src/main/resources/logging.properties

```diff
@@ -0,0 +1,7 @@
+handlers= java.util.logging.FileHandler
+
+# File Logging
+java.util.logging.FileHandler.pattern = %h/.scenebuilder/logs/scenebuilder-${version}.log
+java.util.logging.FileHandler.count = 1
```

Comment:
I see that the log is restarted whenever you stop and start a new instance of Scene Builder, removing the previous logs.
Should we just have a rolling system with a maximum number of files (like 10), and size per file (like 1 MB), but keep everything until the limit is reached?